persistent/interfaces.py

```diff
@@ -23,7 +23,7 @@
     from persistent.cPersistence import UPTODATE
     from persistent.cPersistence import CHANGED
     from persistent.cPersistence import STICKY
-except ImportError: #pragma NO COVER
+except ImportError: # pragma: no cover
```

Comment:
That one in particular is meaningful. The .coveragerc defines the exclude pattern like so:
```
[report]
exclude_lines =
    # pragma: no cover
```

Without that change, the coverage numbers are wrong, and it seemed important to maintain and improve coverage, which is only possible with accurate numbers. I can still back it out if you want.